### PR TITLE
Update German texts

### PR DESCRIPTION
**Pull request by Seelenoede**
As a christmas gift to the community I updated the German texts.
Right now I am a bit unhappy about two things:
1. "wall" I translated it to "Pinnwand", but it doesn't sound nice.
2. Fetishes in the feed. I couldn't create whole sentences with the fetish and the specifics (giving, wearing...). Currently I display it like "Seelentest is into test (giving)" or in German "Seelentest steht auf test (aktiv)".

And I didn't translate Feed because it has a nice sound.
I also didn't follow any discussion regarding the translation of the webpage, so I cannot make promises about consistency.

### CODE REVIEW

**Comment by kistitan** [comment]
Hi @Seelenoede,

I apologize for the late replay.
I started to add a comment here last week, but apparently I did not send it :(

We can't say enough how much we do appreciate external contribution.
Thank you very much for your help.

Please allow as just a quick look from a German speaker before merging your changes.
@rakvat 

Happy new year to you ;)

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1,73 +1,50 @@
...
-  <string name="button_send_description">Nachricht senden</string>
-  <string name="input_message_description">Deine Nachricht</string>
   <string name="menu_friendrequests">Freundschaftsanfragen</string>
+  <string name="menu_introduce">Mein Profil teilen</string>
+  <string name="menu_notifications">Benachrichtigungen</string>
+  <string name="menu_upload_media">Lade Foto hoch</string>
+  <string name="menu_conversations">Unterhaltungen</string>
+  <string name="menu_settings">Einstellungen</string>
+  <string name="menu_logout">Ausloggen</string>
+  <string name="menu_feed">Feed</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: In the web version we translated this to "Aktivitätschronik"

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1,73 +1,50 @@
...
-  <string name="message_media_picture_upload_confirmation">Ich bezeuge, dass dieses Foto von mir ist, oder dass ich die explizite Berechtigung habe, es auf Fetlife zu teilen.\nIch erzähle keine Lügen</string>
-  <string name="button_media_picture_upload_confirmation">Bestätigen</string>
-  <string name="button_media_picture_upload_cancel">Abbrechen</string>
   <string name="title_media_picture_upload_confirmation">Füge dein Bild hinzu</string>
-  <!-- TODO: consider translating text_about -->
 
+  <!-- *** Settings texts *** -->
+
+  <string name="settings_category_general_start">Startbildschrim-Einstellungen</string>
+  <string name="settings_title_general_feed_as_start">Nutze den Feed, als Startbildschirm</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I think there is no `,` here

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -80,12 +57,180 @@
...
+
+
   <string name="settings_title_notification_info_enabled">Info-Benachrichtigungen</string>
   <string name="settings_summary_notification_info_enabled">Aktiviere Benachrichtigungen für allgemeine Informationen</string>
   <string name="noification_title_new_one_or_more_notification">Neue Benachrichtigung</string>
   <string name="noification_text_new_one_or_more_notification">Auf dich warten eine oder mehr Benachrichtigungen</string>
-  <string name="menu_logout">Ausloggen</string>
+  <string name="logon_check_remember_me">Angemeldet bleiben</string>
+  <string name="settings_title_profile">Profileinstellungen</string>
+  <string name="settings_summary_profile">Einstellungen zu deinem FetLife App Profil</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: in the web version we capitalized `du`, `dein`, ...

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="button_delete_user_data_cancel">Abbrechen</string>
+  <string name="button_media_picture_upload_hint_caption">Beschreibung</string>
+  <string name="button_media_picture_upload_check_friends_only">Nur für Freunde sichtbar</string>
+  <string name="message_friend_decision_failed">FetLife App: Deine Entscheidung über die Freundschaft konnte nicht gesendet werden, weil du dich ausgeloggt hast, bevor sie verarbeitet wurde</string>
+
+  <!-- *** Feed texts *** -->
+
+  <string name="title_activity_feed">Feed</string>
+
+  <string name="feed_title_friends">%1$s hat sich mit %2$d Nutzern angefreundet</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: we translated `member` to `Mitglied` in the web version

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="feed_title_new_post">hat ein neues Schriftstück hochgeladen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `post` was tranlated to `Text` in the web-version. You can check the web version at https://github.com/fetlife/translations

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="feed_title_new_posts">postete %1$d neue Schriftstücke</string>
+  <string name="feed_title_new_grouppost">hat einen neuen Gruppenpost hochgeladen</string>
+  <string name="feed_title_new_groupposts">postete %1$d neue Gruppenposts</string>
+  <!-- TODO: need a better word for "wall" than "Pinnwand" -->
+  <string name="feed_title_new_wallpost">hat einen neuen Pinwandpost hochgeladen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: geschrieben?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="feed_title_new_wallposts">postete %1$d neue Pinwandposts</string>
+  <string name="feed_title_group_join">trat einer Gruppe bei</string>
+  <string name="feed_title_group_joins">trat %1$d Gruppen bei</string>
+  <string name="feed_title_new_picture">hat ein neues Foto hochgeladen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: please use `Bild`. It's like that in the web-version and in your next line

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="feed_title_like_unknown">Noch nicht unterstütztes Liebesevent</string>
+  <string name="feed_title_new_friend">hat sich mit einem Mitglied angefreundet</string>
+  <string name="feed_title_new_friends">hat sich mit %1$d Mitgliedern angefreundet</string>
+  <string name="feed_title_new_follow">folgte einem Mitglied</string>
+  <string name="feed_title_new_follows">folgte %1$d Mitgliedern</string>
+  <string name="feed_title_relation_unknown">Noch nicht unterstütztes Beziehungsevent</string>
+
+  <string name="feed_title_rsvp_yes">geht zu einem Event</string>
+  <string name="feed_title_rsvps_yes">geht zu %1$d Events</string>
+  <string name="feed_title_rsvp_maybe">geht vielleicht zu einem Event</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `Event` is `Veranstaltung` in the web-version

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="feed_title_new_statuscomments">hat %1$d Statusupdates kommentiert</string>
+  <string name="feed_title_new_videocomment">hat ein Video kommentiert</string>
+  <string name="feed_title_new_videocomments">hat %1$d Videos kommentiert</string>
+  <string name="feed_title_new_groupcomment">hat einen Gruppenpost kommentiert</string>
+  <string name="feed_title_new_groupcomments">hat %1$d Gruppenposts kommentiert</string>
+  <string name="feed_title_new_status">hat ein Statusupdate hinzugefügt</string>
+  <string name="feed_title_new_statuses">hat %1$d Statusupdates hinzugefügt</string>
+  <string name="feed_title_new_postcomment">hat ein Schriftstück kommentiert</string>
+  <string name="feed_title_new_postcomments">hat %1$d Schriftstücke kommentiert</string>
+  <string name="feed_title_people_into_fetish">hat einen Fetisch</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: maybe "steht auf einen Fetisch"?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 10]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="feed_title_people_into_fetishes">hat %1$d Fetische</string>
+  <string name="feed_caption_member_count" >%1$d Mitglieder</string>
+
+  <string name="settings_title_feed">Feedeinstellungen</string>
+  <string name="settings_summary_feed">Einstellungen, wie die Feed-Geschichten dargestellt werden sollen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: maybe "Einstellungen für die Darstellung der Aktivitätschronik"

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 11]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="settings_category_feed_autoexpand">Auto. Erweiterung der Feed-Geschichten</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: maybe "Einträge in Aktivitätschronik automatisch erweitern"?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 12]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="settings_title_feed_auto_expand_like">Neue Liebe</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: that is plural in the English version. I'm on the fence if "loves" should be translated to "Herzen" here

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 13]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="settings_summary_feed_auto_expand_like">Auto. erweitere Feed-Geschichten, wenn ein gefolgter Nutzer etwas liebte</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: "Mitglied, dem Du folgst"?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 14]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="settings_title_feed_auto_expand_relation">Neue Beziehungen</string>
+  <string name="settings_summary_feed_auto_expand_relation">Auto. erweitere Feed-Geschichten, wenn ein gefolgter Nutzer sich mit jemanden angefreundet hat</string>
+  <string name="settings_title_feed_auto_expand_profile">Profilupdates</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Profilaktualisierungen

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 15]
```diff
@@ -80,12 +57,180 @@
...
+  <!-- I assume that RSVP stands for "Répondez s'il vous plaît" -->
+  <string name="settings_title_feed_auto_expand_event">Eventgeschichten</string>
+  <string name="settings_summary_feed_auto_expand_event">Auto. erweitere Feed-Geschichten, wenn ein Nutzer sich an einem Event interessiert</string>
+  <string name="settings_title_feed_auto_expand_group">Gruppengeschichten</string>
+  <string name="settings_summary_feed_auto_expand_group">Auto. erweitere Feed-Geschichten, wenn ein Nutzer sich an einer Gruppe interessiert</string>
+  <string name="settings_title_feed_auto_expand_media">Neue Medien</string>
+  <string name="settings_summary_feed_auto_expand_media">Auto. erweitere Feed-Geschichten, wenn ein Nutzer ein neues Bild oder Video hochgeladen hat</string>
+
+  <!-- TODO: refractor to whole sentences -->
+  <string name="pojo_value_fetish_status_giving">(aktiv)</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I wonder if this should be translated at all, as the fetish itself will still be English.

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 16]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="pojo_value_fetish_status_receiving">(passiv)</string>
+  <string name="pojo_value_fetish_status_watching">(sehend)</string>
+  <string name="pojo_value_fetish_status_wearing">(tragend)</string>
+  <string name="pojo_value_fetish_status_watching_others_wear">(andere tragen sehen)</string>
+  <string name="pojo_value_fetish_status_everything_to_do_with_it">(alles, was damit zu tun hat)</string>
+  <string name="pojo_value_fetish_activity_into">%1$s steht auf %3$s %2$s</string>
+  <string name="pojo_value_fetish_activity_curious_about">%1$s ist neugierig auf %3$s %2$s</string>
+  <string name="pojo_value_fetish_activity_soft_limit">%3$s %2$s ist ein weiches Limit für %1$s</string>
+  <string name="pojo_value_fetish_activity_hard_limit">%3$s %2$s ist ein hartes Limit for %1$s</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: for -> für

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 17]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="action_clear_notification_history">Lösche alle</string>
+
+
+  <!-- *** Strings for login screen *** -->
+
+  <string name="prompt_username">Benutzername</string>
+  <string name="prompt_password">Passwort</string>
+  <string name="prompt_new_message">Deine Nachricht</string>
+  <string name="action_sign_in_short">Login</string>
+  <string name="error_invalid_username">Dieser Benutzername ist ungültig</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Benutzername -> Name

Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: @kistitan in the web version there is no separate error message for nickname and password for security reasons (make it more difficult to guess passwords).

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 18]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="forgot_login">Logindaten vergessen?</string>
+
+
+  <!-- *** Strings for Friend Requests screen *** -->
+
+  <string name="text_friendrequests_rejected">Abgelehnt</string>
+  <string name="text_friendrequests_accepted">Akzeptiert</string>
+  <string name="text_friendrequests_toaccept">Akzeptieren</string>
+  <string name="text_friendrequests_toreject">Ablehnen</string>
+  <string name="friendrequest_activity_hint">Wische nach rechts zum Akzeptieren, nach links um abzulehnen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `,` after `links`?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 19]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="header_friendrequest">Freundschaftsanfragen</string>
+  <string name="header_friendsuggestion">Geteilte Profile</string>
+
+
+  <!-- *** Strings for making new friends *** -->
+
+  <string name="message_nfc_not_available">Dieses Feature benötigt NFC.</string>
+  <string name="message_android_version_not_sufficient">Dieses Feature benötigt eine neuere Android-Version.</string>
+  <string name="message_nfc_turned_off">Bitte NFC aktivieren um dieses Feature zu nutzen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `,` after `aktivieren`

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 20]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="nfc_tap_your_device">Lege dein Gerät an ein anderes Android-Gerät, um den Beam zu aktivieren</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `Stoße`?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 21]
```diff
@@ -80,12 +57,180 @@
...
+
+
+  <!-- *** Strings for Photo Upload *** -->
+
+  <string name="message_image_upload_started">Beame dein Foto hoch...</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: we used `Bild` before

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 22]
```diff
@@ -80,12 +57,180 @@
...
+  <string name="undo_no_longer_possible">Rückgängig nicht mehr möglich</string>
+
+  <string name="snackbar_version_notification">Im \'Über\'-Fenster kannst du sehen, was neu in v%1$s ist</string>
+
+  <string name="input_message_description">Deine Nachricht</string>
+  <string name="button_new_conversation_discription">Neue Unterhaltung</string>
+  <string name="button_send_description">Nachricht senden</string>
+  <string name="action_settings">Einstellungen</string>
+  <string name="settings_title_general">Allgemeine Einstellungen</string>
+  <string name="settings_summary_general">Überprüfe die allgemeinen Appeinstellungen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: "App-Einstellungen"?

**Comment by Seelenoede** [comment]
@rakvat no problem. I am glad somebody did it.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 20 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I think `Stoße` is not really usable here. To use NFC only the close proximity is important but `Stoße` sounds too harsh.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I am not sure. In the English version there is `uploaded`. `uploaded a new wall post`

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 20 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Not outdated

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 17 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Not really outdated.

I think the problem was also discussed the last time I contributed to the translation.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 15 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I put the fetish related strings into a comment for now.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 12 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: not outdated.

I am not sure about this. I used "liebte" everywhere else "loved" appeared. So it was logical for me to translate "loves" to "Liebe" or "Lieben". I think the singular sounds nicer and is also usable in this context

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: not outdated

**Comment by Seelenoede** [comment]
Please be aware that some comments of @rakvat are not outdated.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 22 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I am not a big fan of dividing words using "-" if the word is readable.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: @kistitan is that a c&p error? should it be `wrote a new wall post`? I'm fine with it, if you want it that way. Just double checking.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 22 above)
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: ok

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (kistitan) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: yep, this should be updated in the English version too

**Comment by kistitan** [comment]
Merging for now, thanks for the help :)
More English text is coming and also the English base is going to be improved so similar help will be more than welcome in the future too :))

**Pull request merged**